tests/test_acceptance.py

```diff
@@ -83,6 +82,13 @@ def purge_output_dirs():
                 id="Nepal project",
                 marks=pytest.mark.skip(reason="WPF Nepal test directory not presnt"),
             ),
+            pytest.param(
+                "bolivia",
+                id="Bolivia project",
+                marks=pytest.mark.skip(
+                    reason="Bolivia project test directory not presnt"
+                ),
```

Comment:
You can remove this marker as the bolivia data is already present in the `svn` repository and should be picked up by our TeamCity agents.